content/en/apps/guides/database/muting_in_dashboards.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Contact Muting Guideline in SQL queries"
+linkTitle: "Contact Muting in SQL queries"
+weight: 
+description: >
+  How to write SQL queries excluding muted contacts correctly
+relevantLinks: >
+---
+
+When a contact gets muted, two of [many things](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#muting) happen:
```

Comment:
```suggestion
When a contact gets muted, two of [many things]({{< ref "apps/reference/app-settings/transitions#muting" >}}) happen: 
```

Reply:
Sorry i missed the earlier comment on ref links